parlai/tasks/interactive/worlds.py

```diff
@@ -16,4 +18,57 @@ class InteractiveWorld(DialogPartnerWorld):
     especially for those cases for given tasks.
     """
 
-    pass
+    def __init__(self, opt, agents, shared=None):
+        super().__init__(opt, agents, shared)
+        self.init_contexts()
+        self.turn_cnt = 0
+
+    def init_contexts(self):
+        """
+        Override to load or instantiate contexts to be used to seed the chat.
+        """
+        pass
+
+    def get_contexts(self):
+        """
+        Override to return a pair of contexts with which to seed the episode.
+
+        This function will be called before the first turn of every episode.
+        """
+        return ['__SILENCE__', '']
```

Comment:
do we want this to be the default? not all datasets have a silence token?

Reply:
yes, i can remove this.